scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -33,8 +33,10 @@ async def _main() -> None:
     start_time = datetime.now()
     print("Executing notebooks:")
     results = await asyncio.gather(*(execute_notebook(job) for job in jobs))
-    print("Checking for trailing jobs...")
-    results.append(cancel_trailing_jobs(start_time))
+
+    if not args.ignore_trailing_jobs:
+        print("Checking for trailing jobs...")
+        results.append(cancel_trailing_jobs(start_time))
```

Comment:
Nit that the blank line was significant
```suggestion
        results.append(cancel_trailing_jobs(start_time))

```